gsk4/src/auto/mod.rs

```diff
@@ -2,13 +2,6 @@
 // from gir-files (https://github.com/gtk-rs/gir-files.git)
 // DO NOT EDIT
 
-#[cfg(any(feature = "broadway", feature = "dox"))]
```

Comment:
Unexpected, you took the from a gtk build without the broadway backend enabled

Reply:
Thanks I didn't noticed. I took it from a personal build. It seems when you last updated gir-files, you didn't update dl.sh script either, I assume you took it from a personal build too, right?

Reply:
No, I use the gnome-nightly flatpak SDK, as it's assured to be built against latest releases/all the expected features enabled. You can enter the container with ` flatpak run --filesystem=home -d --command=sh org.gnome.Sdk//master` and then you can copy the gir files from their usual location at `/usr/share/gir-1.0/` to your home directory.

The dl.sh expects a distro having the latest release, we could maybe use fedora's gtk4-devel packages there but I don't plan on using it for gtk4

Reply:
done in https://github.com/gtk-rs/gir-files/pull/82